src/core/utils/map_to_object.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-export function mapToObject<V = unknown>(map: Map<string, V>) {
+export function mapToObject<V = unknown>(map: ReadonlyMap<string, V>) {
```

Comment:
Does this have any consequence in trying to pass in a Map?

Reply:
This function doesn't modify the argument and `Map` is compatible with `ReadonlyMap` so this change just makes this function more generally useful.